lib/flume/api.ex

```diff
@@ -60,9 +60,13 @@ defmodule Flume.API do
         Config.pipeline_names() |> Enum.map(&pause(&1, temporary))
       end
 
-      defdelegate pause(pipeline_name, temporary \\ true), to: Pipeline.Event
+      def pause(pipeline_name, options \\ Pipeline.Event.default_pause_resume_options()) do
+        apply(Flume.Config.pipeline_api_module(), :pause, [pipeline_name, options])
+      end
 
-      defdelegate resume(pipeline_name, temporary \\ true), to: Pipeline.Event
+      def resume(pipeline_name, options \\ Pipeline.Event.default_pause_resume_options()) do
```

Comment:
Can we think of a better name for this - `Pipeline.Event.default_pause_resume_options()`

E.g - `Pipeline.Event.control_options()`???

`pause` & `resume` are operations that allow a user to control a pipeline!

Also can we move `pause` & `resume` to `Pipeline.Controller` module?

Reply:
> Can we think of a better name for this - Pipeline.Event.default_pause_resume_options()
E.g - Pipeline.Event.control_options()???
pause & resume are operations that allow a user to control a pipeline!

👍 
 > Also can we move pause & resume to Pipeline.Controller module?

`pause/resume` functions in `Flume.API` or the ones in `Flume.Pipeline.Event`?